README.md

```diff
@@ -82,15 +82,60 @@ module.exports = (ctx) => ({
 });
 ```
 
+## Reading config from eik.json or package.json
+
+By default, this plugin will try to read config from your projects `eik.json` or `package.json` files in the root of the current working directory.
+
+The path to the location of an `eik.json` file can be specified with the `path` option.
+`path` defaults to the current working directory.
+
+```js
+module.exports = (ctx) => ({
+    plugins: [
+        require('@eik/postcss-import-map')({ path: '/path/to/eik.json' }),
+        require('postcss-import')({
+            // It needs to be added here as well to ensure everything is mapped
+            plugins: [
+                require('@eik/postcss-import-map')({
+                    path: '/path/to/eik.json',
```

Comment:
@digitalsadhu why are the snippets here using absolute paths, when the API options docs later on use relative paths: `cwd/eik.json`?

Reply:
Don't really have a good answer or reason.